src/app/+visualization/co-investigator-network/co-investigator-network.component.html

```diff
@@ -29,7 +29,7 @@
         <div class="card">
           <div class="card-header">{{ 'VISUALIZATION.NETWORK.COINVESTIGATOR.OBJECTS_PER_YEAR' | translate }}</div>
           <div class="card-body">
-            <table class="table table-sm">
+            <table class="table table-sm" aria-labelledby="objectsPerYear">
```

Comment:
I'm still concerned about these `objectsPerYear` labbeledbys that have no `id` matching `objectsPerYear`. (and similar, on many tables).